src/services/vaultTimeout.service.ts

```diff
@@ -103,13 +107,15 @@ export class VaultTimeoutService implements VaultTimeoutServiceAbstraction {
         }
     }
 
-    async logout(): Promise<void> {
-        // TODO Add logic for loggedOutCallback
+    async logOut(): Promise<void> {
+        if (this.loggedOutCallback != null) {
+            await this.loggedOutCallback();
+        }
     }
 
     async setVaultTimeoutOptions(vaultTimeout: number, vaultTimeoutAction: string): Promise<void> {
         await this.storageService.save(ConstantsService.vaultTimeoutKey, vaultTimeout);
-        // TODO Add logic for vaultTimeoutAction
+        await this.storageService.save(ConstantsService.vaultTimeoutActionKey, vaultTimeoutAction);
         await this.cryptoService.toggleKey();
     }
```

Comment:
Param names seem a little too verbose. I'd suggest just `timeout` and `action`. The method name already tells us this is for settings vault timeout.

Reply:
Updated.